### PR TITLE
Adapt artifacts to new source-provides logic

### PR DESCRIPTION
**Pull request by DeKe42**
- [X] Windows
- [x] Darwin
- [x] Linux

- [x] Clean up, delete unneeded

### CODE REVIEW

**Comment by cugu** [comment]
Solves #10

File: data/linux.yaml [hunk 1]
```diff
@@ -4,8 +4,7 @@ name: AnacronFiles
 doc: Anacron files.
 sources:
 - type: FILE
-  attributes:
-    paths:
+  attributes: paths:
```
Comment (cugu) [inline data/linux.yaml]: No.

File: data/windows.yaml [hunk 2]
```diff
@@ -760,11 +766,15 @@ sources:
     - '\WINNT35'
     - '\WTSRV'
     separator: '\'
+  provides: [{key: environ_systemroot}]
```
Comment (cugu) [inline data/windows.yaml]: Missing regex?

File: data/windows.yaml [hunk 3]
```diff
@@ -790,11 +800,12 @@ sources:
     - '\WINNT35'
     - '\WTSRV'
     separator: '\'
+  provides: [{key: environ_windir}]
```
Comment (cugu) [inline data/windows.yaml]: Missing regex

File: data/windows.yaml [hunk 4]
```diff
@@ -1699,17 +1710,31 @@ sources:
...
   the paths for each profile.
+labels: [Users]
 sources:
 - type: REGISTRY_VALUE
   attributes: {key_value_pairs: [{key: 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\ProfileList\*', value: 'ProfileImagePath'}]}
-labels: [Users]
-provides: [users.sid, users.userprofile, users.homedir, users.username]
+  provides: 
+    - key: users.userprofile
+    - key: users.homedir
```
Comment (cugu) [inline data/windows.yaml]: Define values

File: data/windows.yaml [hunk 5]
```diff
@@ -2895,7 +2920,7 @@ sources:
     key_value_pairs:
       - {key: 'HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\TimeZoneInformation', value: 'StandardName'}
       - {key: 'HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\TimeZoneInformation', value: 'TimeZoneKeyName'}
-provides: [time_zone]
+  provides: [{key: time_zone}]
```
Comment (cugu) [inline data/windows.yaml]: Which key value pair provides this?

File: data/windows.yaml (hunk 2 above)
Comment (DeKe42) [reply data/windows.yaml]: No.

File: data/linux.yaml [hunk 6]
```diff
@@ -4,7 +4,7 @@ name: AnacronFiles
 doc: Anacron files.
 sources:
 - type: FILE
-  attributes:
+  attributes:
```
Comment (cugu) [inline data/linux.yaml]: Remove trailing whitespace

File: data/macos.yaml [hunk 7]
```diff
@@ -937,9 +937,13 @@ doc: Users directories in /Users
 sources:
 - type: DIRECTORY
   attributes: {paths: ['/Users/*']}
+  provides:
+    - key: users.username
+      regex: '/Users/([^/]+)'
+    - key: users.homedir
+      regex: '/Users/([^/]+)'
```
Comment (cugu) [inline data/macos.yaml]: Wrong regex

File: data/windows.yaml (hunk 2 above)
Comment (cugu) [inline data/windows.yaml]: Add 
```
    - key: environ_systemdrive
      regex: '^(..)'
```

**Pull request merged**